capstone/src/main/webapp/createStudySet.html

```diff
@@ -1,21 +1,116 @@
+    />
     <!--Import materialize.css-->
-    <link type="text/css" rel="stylesheet" href="css/materialize.min.css"  media="screen,projection"/>
+    <link
+      type="text/css"
+      rel="stylesheet"
+      href="css/materialize.min.css"
+      media="screen,projection"
+    />
 
     <!--Let browser know website is optimized for mobile-->
-    <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
   </head>
   <body>
-     <h3>Create Study Set</h3>
-     <!--JavaScript at end of body for optimized loading-->
-     <script type="text/javascript" src="js/materialize.min.js"></script>
+    <nav class="nav-wrapper">
+      <a href="#" class="brand-logo">Studentia</a>
+      <a href="#" class="sidenav-trigger" data-target="slide_out"
+        ><i class="material-icons">menu</i></a
+      >
+      <ul id="nav-mobile" class="right hide-on-med-and-down">
+        <li><a href="findStudySet.html">Search</a></li>
+        <li><a href="index.html">Home</a></li>
+      </ul>
+    </nav>
+
+    <div class="row">
+      <div class="col s4 left-align teal lighten-4 z-depth-3">
+        <p class="flow-text white-text">First Step to Create Study Set</p>
+        <form action="/study_set" method="POST">
+          <div class="input-field col s12">
+            <input type="text" id="user_id" name="user_id" />
+            <label for="user_id">user_id:</label>
+          </div>
+
+          <div class="input-field col s12">
+            <input type="text" id="title" name="title" />
+            <label for="title">title:</label>
+          </div>
+
+          <div class="input-field col s12">
+            <label for="subject">subject:</label>
+            <input type="text" id="subject" name="subject" />
+          </div>
+
+          <div class="input-field col s12">
+            <label for="description">description:</label>
+            <input type="text" id="description" name="description" />
+          </div>
+
+          <div class="input-field col s12">
+            <label for="university_id">university_id:</label>
+            <input type="text" id="university_id" name="university_id" />
+          </div>
+
+          <div class="input-field col s12">
+            <label for="professor">professor:</label>
+            <input type="text" id="professor" name="professor" />
+          </div>
+
+          <div class="input-field col s12">
+            <label for="academic_time">academic_time:</label>
+            <input type="text" id="academic_time" name="academic_time" />
+          </div>
+
+          <div class="input-field col s12">
+            <label for="course_name">course_name:</label>
+            <input type="text" id="course_name" name="course_name" />
+          </div>
+
+          <div class="input-field col s12">
+            <label for="creation_time">creation_time:</label>
+            <input type="text" id="creation_time" name="creation_time" />
```

Comment:
I don't think this should also be on the page directly, there are methods in JS and Java to get the current date and time. This will avoid unneccesary errors.